tests/plotting/test_plotting.py

```diff
@@ -1606,7 +1606,7 @@ def test_link_views_camera_set(sphere, verify_image_cache):
 
 
 def test_orthographic_slicer(uniform):
-    uniform.set_active_scalars('Spatial Cell Data')
+    uniform.set_active_scalars('Spatial Point Data')
```

Comment:
@banesullivan Could you explain the reason why we failed the test in Cell Data?

Reply:
This was an artifact of me figuring out the composite rendering differences. I reverted